examples/next/src/components/AvatarDemo.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import { useState } from "react";
+
+import { Avatar, Button, Flex } from "@animareflection/ui";
+
+import { Wrapper } from "components";
+
+const AvatarToggleDemo = () => {
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  return (
+    <Wrapper title="Avatar">
+      <Flex direction="column" gap={2}>
+        {isLoaded ? (
+          <Avatar src="/img/logo.png" alt="avatar" />
+        ) : (
+          <Avatar src="" alt="" />
+        )}
```

Comment:
This could be simplified to checking `isLoaded` in the props themselves, that way there wouldn't need to be 2 `<Avatar ...` lines